### ISSUE TITLE
[BUG] RKE1 Clusters taking hours to come back up after a migration via backup/restore operator

### ISSUE THREAD

**Issue by nickwsuse** [rancher/rancher#40681]
**Rancher Server Setup**
- Rancher version: `v2.6-head Commit ID: 240554b`
- Installation option (Docker install/Helm Chart): Helm
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): RKE1
- Proxy/Cert Details: byo-valid

**Information about the Cluster**
- Kubernetes version: `v1.24.10`
- Cluster Type (Local/Downstream): Downstream
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider): EKS

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom): Admin

**Describe the bug**
When doing a migration of Rancher from one instance to another, the RKE1 clusters are taking significantly longer to come up than they have previously. Normally it would take around 5-10 minutes after the restore for the RKE1 cluster to be back up, but now it's taking hours to come up.

It does seem to vary as I've seen it come back up anywhere from 30 minutes after restore, to over 5 hours (probably longer than that, but 5 hours was the last check I did before going to bed)

**To Reproduce**
1. Create a v2.6-head Rancher instance (this will repro on any version it seems, but this version has taken the longest so far)
2. Go into the local cluster > Storage > Secrets
3. Create a new opaque secret with your S3 credentials (where the backup file will be stored)
4. Go to local > Apps > Rancher Backups
5. Install Rancher Backups with either v2.1.4 or v2.1.5-rc2 (if you've enabled pre-release versions on the Rancher instance)
6. Create a downstream EKS RKE1 cluster with k8s v1.24.10
7. Create a downstream EKS RKE2 cluster with k8s v1.24.10+rke2r1
8. Once the clusters are provisioned, go to local > Rancher Backups > Backups and create a backup
9. Begin the migration process ([noted here](https://confluence.suse.com/pages/viewpage.action?pageId=1050509654))
10. Once migrated, wait for the clusters to come back up

**Result**
It takes a very long time for the RKE1 cluster to come back up

**Expected Result**
It takes 10 minutes or less for the RKE1 cluster to come back up

**Screenshots**
Cluster Management Page - Five Hours and Cluster Is Not Up
![image](https://user-images.githubusercontent.com/107147272/221270748-996b9a26-5438-49d8-9656-e14b48c54ef3.png)

**Additional context**
Quite possibly related to https://github.com/rancher/rancher/issues/40080 but making this ticket in case it's a separate issue

**Comment by Oats87** [comment]
Currently waiting on @nickwsuse to provide an environment where this has occurred so I can debug it. Otherwise, when I free up, I can attempt to reproduce it on my own.

**Comment by snasovich** [comment]
@nickwsuse (cc @ronhorton @sowmyav27 ), moved this one to "To Test" to see if it's in fact reproducible - I don't believe it was reproduced lately and could be related to testing steps where the same backup is reused for multiple restores.

**Comment by nickwsuse** [comment]
I've not encountered this issue at all in the last three weeks or so. I think it's safe to say it's no longer an issue so I'll close the ticket.
